website/src/blogs/becoming-a-kubernauttop-learnings-in-my-journey-of-becoming-a-cka.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Becoming a Kubernaut: Top Learnings in my journey of becoming a CKA"
+title: Becoming a Kubernaut - Top Learnings in my journey of becoming a CKA
```

Comment:
The title of blog was showing only half `Becoming a Kubernaut:` the rest got trimmed because of the `:` in the title. Not using columns in title will be good